transbank/webpay/webpay_plus/__init__.py

```diff
@@ -5,4 +5,6 @@
 mall_default_child_commerce_codes  = ['597055555536', '597055555537']
 default_api_key = "579B532A7440BB0C9079DED94D31EA1615BACEB56610332264630D42D0A36B1C"  # Default API_KEY for Webpay
 default_integration_type = IntegrationType.TEST  # Default integration_type is TEST
-webpay_plus_deferred_commerce_code = "597055555540"
\ No newline at end of file
+webpay_plus_deferred_commerce_code = "597055555540"
+webpay_plus_mall_deferred_default_commerce_code = "597055555544"  # Default COMMERCE_CODE for Webpay Plus Mall Deferred
```

Comment:
The comment in this line is redundant, sice the variable name is enoug to understand that this is the default comerce code for webpay plus mall deferred.